.github/workflows/claude.yml

```
@@ -0,0 +1,64 @@
+name: Claude Code
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)